starters/basic-starter/cypress.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from "cypress";
```

Comment:
We should avoid adding files to the `starters/basic-starter` directory since this would end up on the apps that folks are building. I think we everything related to the tests should be kept outside here.

Reply:
Only problem with that is I need a `tsconfig.json` in the root directory, so that would block this on @harumijang's PR to add linting.

Reply:
...on the other hand, that PR got merged, apparently! So I'll try to move all this stuff to the root.